gcp_variant_transforms/testing/integration/bq_to_vcf_tests/no_options.json

```diff
@@ -3,6 +3,7 @@
     "test_name": "bq-to-vcf-no-options",
     "input_table": "gcp-variant-transforms-test:bq_to_vcf_integration_tests.4_0__suffix",
     "output_file_name": "bq_to_vcf_no_options.vcf",
+    "use_1_based_coordinate": false,
```

Comment:
I am expecting this test (and all the following tests) should have failed. If not something is wrong.

s/`use_1_based_coordinate `/`bq_uses_1_based_coordinate`